accepted/PSR-17-http-factory-meta.md

```diff
@@ -290,6 +290,12 @@ responsible for the creation of the PHP resource to use as the basis for the
 stream, and therefore also assumes responsibility for the read/write pointer
 state of the resource and resulting stream.
 
+And finally, the instances created by the `RequestFactoryInterface()`,
+`ResponseFactoryInterface()` and `ServerRequestFactoryInterface` factories
```

Comment:
These are not functions.

Reply:
It says "factories", not "functions".

There's only one function in each interface - spelling out the method-names as well seemed a bit excessive?

I'll change it if you insist though.

Reply:
I mean `RequestFactoryInterface()` (etc) should be `RequestFactoryInterface`.

Reply:
Oh whoops 😂